actions/actions.py

```diff
@@ -135,14 +237,15 @@ def list_backups() -> None:
     function_set({'message': message})
 
 
-def delete_backup() -> None:
-    backup_name: str = function_get('name')
-    delete_all: bool = function_get('all')
+def delete_backup():
+    """Implementation of easyrsa 'delete-backup' action"""
+    backup_name = function_get('name')
```

Comment:
Please validate input as suggested in the `restore` action.

Reply:
(please see my concerns in the `restore` action reply)

Reply:
Cheers. I suggested the use of `os.path.basename` to avoid path escalation. But I'm OK with allowing any file name.